### PR TITLE
LP-2298 Fix all pep8 linting errors

### PR DESCRIPTION
**Pull request by asadali145**
### Description

[LP-2298](https://philanthropyu.atlassian.net/browse/LP-2298)

Original branch had some conflicts so creating this new PR. Old [PR](https://github.com/philanthropy-u/edx-platform/pull/1862).

### CODE REVIEW

**Comment by asadali145** [comment]
Please ignore pylint and isort. They were out of scope for this ticket.

File: common/djangoapps/philu_commands/helpers.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def generate_course_structure(course_key):
 
             if (curr_block.category == 'discussion' and
                 hasattr(curr_block, 'discussion_id') and
```
Comment (MunamMubashir-arbisoft) [inline common/djangoapps/philu_commands/helpers.py:28]: Can we format this better as well?

**Review by javeriaejaz-arbisoft — APPROVED** [review]

**Review by MunamMubashir-arbisoft — APPROVED** [review]
👍🏼

File: common/djangoapps/philu_commands/helpers.py (hunk 1 above)
Comment (asadali145) [reply common/djangoapps/philu_commands/helpers.py]: @MunamMubashir-arbisoft Can you please have one final look at this change? I have updated this condition. Just have a look at the latest commit.

**Pull request merged**